### PR TITLE
November release post

### PR DESCRIPTION
**Pull request by adrianhall**
Created new post and removed the old preview posts.

### CODE REVIEW

File: _posts/2019-11-11-november-ga-release.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Azure SDK Releases (November 2019)
```
Comment (weshaggard) [inline _posts/2019-11-11-november-ga-release.md]: Should we consider just adding an index page to the release page and have that be the root landing page?

File: _posts/2019-11-11-november-ga-release.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+date: 11 Nov 2019
+sidebar: releases_sidebar
+repository: azure/azure-sdk
+---
+
+The Azure SDK team is pleased to announce the Novemeber 2019 release of the Azure client libraries.  We have some libraries that have reached General Availability and others that we have provided a new preview.
+
+The following libraries have reached our general availability level.  This requires support from all four languages (.NET, Java, Python, and TypeScript).
+
+* Cosmos
```
Comment (kaerm) [inline _posts/2019-11-11-november-ga-release.md]: Cosmos v4 is not GA no?

File: _posts/2019-11-11-november-ga-release.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+* Storage (Blobs, Queues)
+
+In addition, we have made the Azure Core and Identity libraries generally available.
+
+We also updated the following libraries with a new preview:
+
+* App Configuration
+* Event Hubs
+* KeyVault (Certificates)
+* Storage (Files)
```
Comment (kaerm) [inline _posts/2019-11-11-november-ga-release.md]: File-shares at thins point I think

File: _posts/2019-11-11-november-ga-release.md (hunk 2 above)
Comment (mayurid) [reply _posts/2019-11-11-november-ga-release.md]: Right, Cosmos is not GA yet

**Pull request merged**